.travis.yml

```diff
@@ -19,6 +19,8 @@ env:
   - PYTHON_VERSION=3.6 TESTCMD=" -vv --with-timer --with-coverage --cover-package=pymc3 -e test_examples -e test_distributions"
   - PYTHON_VERSION=3.6 TESTCMD=" -vv --with-timer --with-coverage --cover-package=pymc3 pymc3.tests.test_distributions pymc3.tests.test_examples"
   - PYTHON_VERSION=3.6 TESTCMD=" -vv --with-timer --with-coverage --cover-package=pymc3 pymc3.tests.test_distributions_random"
+  - THEANO_FLAGS='floatX=float32'
```

Comment:
We need all the above test runs, and add a new flag, e.g. THEANO_FLOAT_TYPE. Then in test.sh we need to fish out and add it to THEANO_FLAGS.